### ISSUE TITLE
Login breaks with Pyramid 1.2

### ISSUE THREAD

**Issue by naktinis** [bbangert/velruse#82]
Login breaks with Pyramid 1.2. It appears that Pyramid expects the provider login view method to have a different number of arguments. Which version of Pyramid did you test your examples against?

```
File "<...>/pyramid-1.2-py2.6.egg/pyramid/config/views.py", line 319, in viewresult_to_response
    result = view(context, request)
TypeError: login() takes exactly 2 arguments (3 given)
```

**Comment by mmerickel** [comment]
I've been doing tests with 1.3. It would help if your traceback actually included velruse code in it. Anyway I presume the problem is because I'm using a bound method as a view.

**Comment by mmerickel** [comment]
I found the issue at Pylons/pyramid#461. I'll investigate adding this fix.

**Comment by mmerickel** [comment]
fixed in 75edbebea3a0f99486894e8104a4296bdeb4dad1

**Comment by naktinis** [comment]
Thanks!
